### PR TITLE
[Docs] add new yaml files that follow pai protocol

### PR DESCRIPTION
**Pull request by abuccts**
Part of #2260. Add new protocol marketplace for protocol yaml files.

### CODE REVIEW

File: protocol-marketplace/tensorflow-cifar10.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        --variable_update=parameter_server
+        --ps_hosts=$PAI_TASK_ROLE_ps_server_HOST_LIST
+        --worker_hosts=$PAI_TASK_ROLE_worker_HOST_LIST
+        --task_index=$PAI_CURRENT_TASK_ROLE_CURRENT_TASK_INDEX
+        --data_name=<% $data.name %>
+        --data_dir=$PAI_WORK_DIR/data_<% $data.name %>
+        --train_dir=$PAI_WORK_DIR/output_<% $output.name %>
+        --model=<% $parameters.model %>
+        --batch_size=<% $parameters.batchsize %>
+  ps_server:
```
Comment (fanyangCS) [inline protocol-marketplace/tensorflow-cifar10.yaml]: I assume by now there are few people using ps mode to train a distributed TF work? maybe we should remove this example and add MPI example (Horovod) later?

Comment (abuccts) [reply protocol-marketplace/tensorflow-cifar10.yaml]: ok

File: protocol-marketplace/caffe-mnist.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+protocolVersion: 2
```
Comment (sterowang) [inline protocol-marketplace/caffe-mnist.yaml]: Isn't "marketplace/v2" a more aligned folder path to echo with "*/v2" in PR #2260?

Comment (abuccts) [reply protocol-marketplace/caffe-mnist.yaml]: Because the original marketplace serve the whole dir, so I added a new separate directory.

Comment (abuccts) [reply protocol-marketplace/caffe-mnist.yaml]: renamed to `marketplace-v2`

**Comment by abuccts** [comment]
For `commands` field in `taskRoles`, it's an array type, so user can write multiple command items or concatenate several commands using `&&` as one item. It depends on user himself.

For `preCommands` and `postCommands` fields in `deployments`, they are dynamic and related to runtime clusters instead of the job itself, so we separate a `deployments` field to achieve this by design.

**Pull request merged**